.github/workflows/release-python-runtime.yml

```diff
@@ -27,9 +27,55 @@ jobs:
           R2_ACCOUNT_ID: ${{ secrets.PYODIDE_CAPNP_R2_ACCOUNT_ID }}
           R2_ACCESS_KEY_ID: ${{ secrets.PYODIDE_CAPNP_R2_ACCESS_KEY_ID }}
           R2_SECRET_ACCESS_KEY: ${{ secrets.PYODIDE_CAPNP_R2_SECRET_ACCESS_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why does this need a GITHUB_TOKEN as well?

Reply:
Uh... yeah it doesn't.